helm/botkube/values.yaml

```diff
@@ -53,11 +55,13 @@ rbac:
   # -- Use this to create additional RBAC resources for specified groups.
   groups:
     - name: &static-group-name "botkube-plugins-default"
+      create: true
       rules:
         - apiGroups: ["*"]
           resources: ["*"]
           verbs: ["get", "watch", "list"]
     - name: &argocd-group "argocd"
+      create: false # change to true when using ArgoCD plugin
```

Comment:
I think this part is important, but I think inline (the comment which beside the config line) does not reflects helm chart readme. Do you think we should make it like 
```yaml
....
## -- change to true when using ArgoCD plugin
create: false
...
```

Reply:
Agree, pushed fix: https://github.com/kubeshop/botkube/pull/1220/commits/b464ea8c71caaf644e99f9183d3de6edc90381c1